internal/daemonset/testdata/test_worker_daemonset.yaml

```
@@ -26,9 +26,17 @@ spec:
             - matchExpressions:
               - key: node-role.kubernetes.io/master
                 operator: DoesNotExist
+              - key: beta.kubernetes.io/os
```

Shouldn't we use the GA label?

```suggestion
              - key: kubernetes.io/os
```

---

yes, we should. Fixed it